_source/_includes/setup/integrator-cli.md

```diff
@@ -0,0 +1,185 @@
+{%- if page.path contains '.adoc' -%}{% assign adoc = true %}{%- endif -%}
+{%- capture adminLink %}
+  {%- if adoc -%}https://developer.okta.com/login[Okta org]
+  {%- else -%}[Okta org](https://developer.okta.com/login)
+  {%- endif -%}
+{%- endcapture -%}
+
+{% if include.install != "false" %}
+  {%- if include.signup == "false" -%}Sign in to your {{ adminLink }}.
+  {%- else -%}Before you begin, you'll need a free Okta Integrator Free Plan account. To get one, sign up for an {{ adminLink }}. Then, sign in to your {{ adminLink }}.
+  {%- endif -%}
+{% endif %}
+
+{%- if include.type == "spa" -%}
+  {%- assign parts = include.loginRedirectUri | split: '/callback' -%}
+  {%- assign baseUrl = parts[0] -%}
+{%- endif -%}
+
+{%- if include.type == "web" -%}
+  {%- assign parts = include.loginRedirectUri | split: '/authorization-code/callback' -%}
+  {%- assign baseUrl = parts[0] -%}
+{%- endif -%}
+
+
+{% if include.install == "false" %}In the Admin Console:{% else %}Then, in the Admin Console:{% endif %}
```

Comment:
```suggestion
{% if include.install == "false" %}In the Admin Console:{% else %}In the Admin Console:{% endif %}
```

You might be able to remove the install ifelse here? But for now fine with duplication. It doesn't read right to have so many "then" in a row.